### ISSUE TITLE
Why no <title> tag inside page source?

### ISSUE THREAD

**Issue by Atif-Bashir-1998** [inertiajs/inertia-laravel#447]
I am using Inertia's `Head` component to have the `<title>` and `<meta>` tags of my website. I am using Laravel 9 with Inertia (Vue) with SSR enabled. But the issue is that when I view the page-source code, I do not see any `<title>` tag there. What is the reason for this? And will it affect my website's SEO?

Secondly, I am getting `Ziggy` routes of my app in the page source-code. Is there a way to not show them in the page source?

**Comment by usmanvaince** [comment]
Hard refresh and then check your page source, it would be there if SSR is implemented. If you don't refresh your browser, SSR works behind the scene.

**Comment by Atif-Bashir-1998** [comment]
Instead of using inertia's `Head`component to load title and meta tags of different pages, I am rendering the page title and meta inside the `app.blade.php` and it works fine. Here is my updated `app.blade.php` file.

```<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">

	<title>{{ $page['props']['seo']->title }}</title>
        <meta name="description" content="{{ $page['props']['seo']->description }}">

        @inertiaHead

        <!-- Fonts -->
        <link rel="stylesheet" href="https://fonts.bunny.net/css2?family=Nunito:wght@400;600;700&display=swap">

        <!-- Scripts -->
        {!! $page['props']['scripts'] !!}

        @routes
        @vite('resources/js/app.js')
    </head>
    <body class="font-sans antialiased">
        @inertia
    </body>
</html>
```

Here is the controller code for rendering a page:
```
return Inertia::render('Home', [
	'seo' => Seo::select('title', 'description')->where('page_name', 'Home')->first()
]);
```

**Comment by Atif-Bashir-1998** [comment]
> Hard refresh and then check your page source, it would be there if SSR is implemented. If you don't refresh your browser, SSR works behind the scene.

sorry this doesn't work

**Comment by usmanvaince** [comment]
Ok so you are using it inside of app.blade.php and it worked?
Great.!

But do you still need SSR as now you are managing it via app.blade.php

**Comment by usmanvaince** [comment]
My question here is that if we don't put this tag
 @inertiaHead.
 Does it affect the SEO if we are only concerned about the page title and meta tags not about the other rendered html?

**Comment by Atif-Bashir-1998** [comment]
@usmanvaince I am not good at SEO and I don't know if it affects the SEO or not. My SEO team is happy, so I think it is all good :)

**Comment by usmanvaince** [comment]
Let me rephrase my question, if we don't run SSR and just do what you did in the app.blade.php as we are also concerned mainly about meta tags and page titles, on every page change, does browser render the page , or still we have to run the node server. 

Any idea about it ?

**Comment by Atif-Bashir-1998** [comment]
The browser renders the page without running the node server
